src/Illuminate/Database/Eloquent/Relations/Relation.php

```diff
@@ -273,6 +280,26 @@ public function wrap($value)
     }
 
     /**
+     * Set the morph map for polymorphic relations.
+     *
+     * @param  array|null $map
+     * @param  bool $merge
+     * @return array
+     */
+    public static function morphMap($map = null, $merge = true)
+    {
+        if (is_array($map)) {
```

Comment:
@GrahamCampbell Would that imply getting rid of the `is_array` check? Or do you merely mean a return inside the if statement to avoid the else?